rpcs3/Emu/Cell/PPUTranslator.cpp

```diff
@@ -951,7 +951,14 @@ void PPUTranslator::VMADDFP(ppu_opcode_t op)
 		if (data == v128::from32p(1u << 31))
 		{
 			set_vr(op.vd, vec_handle_result(a * c));
-			ppu_log.notice("LLVM: VMADDFP with 0 addend at [0x%08x]", m_addr + (m_reloc ? m_reloc->addr : 0));
+			ppu_log.notice("LLVM: VMADDFP with -0 addend at [0x%08x]", m_addr + (m_reloc ? m_reloc->addr : 0));
+			return;
+		}
+
+		if (data == v128{})
+		{
+			set_vr(op.vd, vec_handle_result(a * c + fsplat<f32[4]>(0.f)));
+			ppu_log.notice("LLVM: VNMSUBFP with -0 addend at [0x%08x]", m_addr + (m_reloc ? m_reloc->addr : 0));
```

Comment:
copy paste error, should be mentioning VMADDFP

Reply:
Foxed.